pkgs/development/python-modules/pybtex/default.nix

```
@@ -1,11 +1,16 @@
-{ lib, buildPythonPackage, fetchPypi, latexcodec, pyyaml }:
+{ lib, buildPythonPackage, fetchPypi
+, latexcodec, pyyaml, setuptools }:
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, latexcodec
, pyyaml
, setuptools
}:
```

It's the code style we use these days. It enables easier parseable diffs and it touches less lines in a change, so merges are less prone to conflicts.